lambdas/DumpSnippets/lambda_function.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+import boto3
+import zipfile
+import base64
+
+# Initialization
+s3 = boto3.client('s3', 'us-west-2')
+try:
+    bn = os.environ['BucketName']
+except KeyError as error:
+    print 'Must specify "BucketName" env var!'
+    raise error
+try:
+    zb = os.environ['ZipsBucket']
```

Comment:
It isn't defined anywhere right now, so it [defaults](https://github.com/maryvilledev/codesplain-lambdas/pull/122/files/449b6e13cdbb631a60faa8a45d91dea0242e5422#diff-7413670d107269ed96b06b5c11a3569cR17) to `codesplain-zips`. But the logic exists for us to define a different bucket if we ever want to.